src/adr/0040-generalize-event-bus.md

```diff
@@ -0,0 +1,27 @@
+#40. Generalize EventBus
+
+Date: 2020-06-11
+
+## Status
+
+Proposed
+
+## Context
+
+When searching for users by quota ratio if we set the value of the parameters to 0, the search feature is supposed to return newly created users
+who have not received any email yet at that point. However, this is not the case due to the quotas are currently being initialized only after
```

Comment:
```suggestion
who have not received any email yet at that point. However, this is not the case because the quotas are currently being initialized only after
```